9.4/root/usr/bin/run-postgresql

```diff
@@ -2,8 +2,9 @@
 
 set -eu
 
-source ${HOME}/common.sh
+source "$CONTAINER_SCRIPTS_PATH"/common.sh
```

Comment:
i'm not bash expert but shouldn't this be `source "${CONTAINER_SCRIPTS_PATH}/common.sh"` ?